datasets/ljspeech.py

```diff
@@ -23,7 +23,7 @@ def build_from_path(in_dir, out_dir, num_workers=1, tqdm=lambda x: x):
   executor = ProcessPoolExecutor(max_workers=num_workers)
   futures = []
   index = 1
-  with open(os.path.join(in_dir, 'metadata.csv'), encoding="utf-8") as f:
+  with open(os.path.join(in_dir, 'metadata.csv'), 'rb') as f:
```

Comment:
I think the recommended way of doing this in Python 2 is to use io.open. Then you can continue to pass an encoding.
https://mail.python.org/pipermail/python-list/2015-March/687124.html

Reply:
I have a problem with launching your system with your trained model checkpoint. Please help me! I described my problem here: https://github.com/keithito/tacotron/issues/94 Thanks